# result: runtime error
def a = [1, 2, 3]
a.iterator_value(4) # expect: Iterator out of bounds.
